multiqc/modules/quast/quast.py

```diff
@@ -108,79 +122,83 @@ def parse_quast_log(self, f):
     def quast_general_stats_table(self):
         """ Take the parsed stats from the QUAST report and add some to the
         General Statistics table at the top of the report """
-
+        
         headers = OrderedDict()
         headers['N50'] = {
-            'title': 'N50 (Kbp)',
+            'title': 'N50 ({})'.format(self.contig_length_suffix),
             'description': 'N50 is the contig length such that using longer or equal length contigs produces half (50%) of the bases of the assembly (kilo base pairs)',
             'min': 0,
-            'suffix': 'bp',
+            'suffix': self.contig_length_suffix,
             'scale': 'RdYlGn',
-            'modify': lambda x: x / 1000
+            'modify': lambda x: x / self.contig_length_multiplier
         }
         headers['Total length'] = {
-            'title': 'Length (Mbp)',
+            'title': 'Length ({})'.format(self.total_length_suffix),
             'description': 'The total number of bases in the assembly (mega base pairs).',
             'min': 0,
-            'suffix': 'bp',
+            'suffix': self.total_length_suffix,
             'scale': 'YlGn',
-            'modify': lambda x: x / 1000000
+            'modify': lambda x: x / self.total_length_multiplier
         }
         self.general_stats_addcols(self.quast_data, headers)
 
     def quast_table(self):
         """ Write some more statistics about the assemblies in a table. """
-
         headers = OrderedDict()
+        
         headers['N50'] = {
-            'title': 'N50 (Kbp)',
-            'description': 'N50 is the contig length such that using longer or equal length contigs produces 50% of the bases of the assembly (kilo base pairs)',
+            'title': 'N50 ({})'.format(self.contig_length_suffix),
+            'description': 'N50 is the contig length such that using longer or equal length contigs produces half (50%) of the bases of the assembly.',
             'min': 0,
-            'suffix': 'bp',
+            'suffix': self.contig_length_suffix,
             'scale': 'RdYlGn',
-            'modify': lambda x: x / 1000
+            'modify': lambda x: x / self.contig_length_multiplier
```

Comment:
These need to be switched to multiply instead of divide. Otherwise the default of dividing by 0.000001 will make the counts a million times bigger ;)